Add missing meta descriptions (#488)
mongodb/docs-golang: source/fundamentals/crud/read-operations/text.txt

Search Text
===========

.. meta::

>    :description: Learn how to perform text searches in MongoDB using Go, including creating text indexes and sorting results by relevance.

Reviewer: ```suggestion
   :description: Learn how to perform text searches with the MongoDB Go Driver, including creating text indexes and sorting results by relevance.
```
